webapp/mixins/scrollToAnchor.js

```diff
@@ -0,0 +1,23 @@
+function scrollToAnchor(anchor, { checkAnchor, $scrollTo }) {
```

Comment:
> [<img alt="vbelolapotkov" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/vbelolapotkov) **Authored by [vbelolapotkov](https://github.com/vbelolapotkov)**
_<time datetime="2019-10-01T10:42:34Z" title="Tuesday, October 1st 2019, 12:42:34 pm +02:00">Oct 1, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,20 @@
+import { scrollTo } from 'vue-scrollto'
```


@roschaefer I like the module and how it works. 👍